docs/visualize/vega.asciidoc

```diff
@@ -279,19 +296,19 @@ to your kibana.yml file.
 [[vega-useful-links]]
 === Useful Links
 
-* https://vega.github.io/editor/[Editor] - includes examples for Vega &
-VegaLite, but does not support any Kibana-specific features like
+* The https://vega.github.io/editor/[Vega Editor] includes examples for Vega &
+Vega-Lite, but does not support any Kibana-specific features like
 Elasticsearch requests and interactive base maps.
-* VegaLite
-https://vega.github.io/vega-lite/tutorials/getting_started.html[Tutorials],
+* Vega-Lite
+https://vega.github.io/vega-lite/tutorials/getting_started.html[tutorials],
 https://vega.github.io/vega-lite/docs/[docs], and
 https://vega.github.io/vega-lite/examples/[examples]
-* Vega https://vega.github.io/vega/tutorials/[Tutorial],
+* Vega https://vega.github.io/vega/tutorials/[tutorials],
```

Comment:
I think we should also add Vega blog posts section here -  I think I wrote 3 or 4 blog posts by now, some  of which may be useful to people.

Reply:
I reorganized this into sections, and added a section with the three blog posts that I found - let me know if there are others.